src/test/java/org/llorllale/cactoos/matchers/HasValuesMatchingTest.java

```diff
@@ -27,16 +27,16 @@
 
 package org.llorllale.cactoos.matchers;
 
+import java.io.IOException;
 import org.cactoos.list.ListOf;
-import org.hamcrest.Description;
-import org.hamcrest.StringDescription;
-import org.hamcrest.core.IsEqual;
+import org.cactoos.text.Joined;
 import org.junit.Test;
 
 /**
  * Test case for {@link HasValuesMatching}.
  *
  * @since 1.0.0
+ * @checkstyle JavadocMethodCheck (500 lines)
```

Comment:
@victornoel I think it's preferable to add Javadoc for the new test (the old ones had it), instead of waiving the requirement.

Reply:
@scristalli I don't agree, each test has already a description via its name + via the `Assertion` first parameter which usually repeat exactly the same thing.
In all cactoos and similar project, we ignore this rule on tests because it's a bother, it's useless and it endanger maintainability :)